kafka/tests/runners.py

```diff
@@ -0,0 +1,130 @@
+# (C) Datadog, Inc. 2018-present
```

Comment:
```suggestion
# (C) Datadog, Inc. 2020-present
```

Reply:
I ended up just getting rid of all these kafkaconsumer files I had. They were from when I was trying to create metrics yesterday that were rate metrics ( and not collectable )

Reply:
👍